app/src/app/[lng]/data/[step]/SubsectorDrawer.tsx

```diff
@@ -549,26 +587,28 @@ export function SubsectorDrawer({
                               gpcReferenceNumber={scope.gpcReferenceNumber!}
                               emissionsFactors={emissionsFactors!}
                             />
-                          </AccordionPanel>
-                        </AccordionItem>
-                      ))}
-                    </Accordion>
-                    <Box w="full" className="grow flex flex-col">
-                      <Box className="grow" />
-                      <Button
-                        onClick={handleSubmit(onSubmit)}
-                        isLoading={isSubmitting}
-                        type="submit"
-                        formNoValidate
-                        w="full"
-                        h={16}
-                        mb={12}
-                        mt={6}
-                      >
-                        {t("add-data")}
-                      </Button>
-                    </Box>
-                  </form>
+                            <Box w="full" className="grow flex flex-col">
+                              <Box className="grow" />
+                              <Button
+                                onClick={handleSubmit(onSubmit)}
```

Comment:
Do you need to pass in the currently selected scope (subcategory) ID here, so the others are ignored?